harvester_e2e_tests/integrations/test_3_vm_functions.py

```diff
@@ -174,6 +174,15 @@ def unset_cpu_memory_overcommit(api_client):
     api_client.settings.update('overcommit-config', spec)
 
 
+@pytest.fixture
+def machine_types(request, api_client):
+    major, minor, micro = api_client.cluster_version.release
+    if major > 1 or minor > 2 or (minor == 2 and micro > 1):
```

Comment:
A ignorable corner case is `0.3.0` 
```
>>> major, minor, micro = 0, 3, 0
>>> major > 1 or minor > 2 or (minor == 2 and micro > 1)
True
```

However, it think it will be more concrete if using `version_check` or `parse_version` explicitly.

Reply:
Yeah but versions before v1.0.0 already EOL, I think we don't need to consider that.

If we use `parse_version` to check versions, the code will longer than this solution, as this might possibly happen in future, I thought this solution would better than use `parse_version`.